src/xlsx/functions/charts.ts

```diff
@@ -541,6 +545,99 @@ function addComboChart(chart: ExcelChartDefinition): XMLString {
   `;
 }
 
+function addPyramidChart(chart: ExcelChartDefinition): XMLString {
+  const dataSets = chart.dataSets;
+  const dataSetsColors = dataSets.map((ds) => ds.backgroundColor ?? "");
+  const colors = new ColorGenerator(dataSets.length, dataSetsColors);
+  const leftDataSet = dataSets[0];
+  const rightDataSet = dataSets[1];
+  const firstColor = toXlsxHexColor(colors.next());
+  const secondColor = toXlsxHexColor(colors.next());
+  const leftBarDataSetNode: XMLString = escapeXml/*xml*/ `
+  <c:ser>
+    <c:idx val="0"/>
+    <c:order val="0"/>
+    <c:invertIfNegative val="0" />
+    ${extractDataSetLabel(leftDataSet.label)}
+    ${shapeProperty({
+      backgroundColor: firstColor,
+      line: { color: firstColor },
+    })}
+    ${chart.labelRange ? escapeXml/*xml*/ `<c:cat>${stringRef(chart.labelRange)}</c:cat>` : ""}
```

Comment:
```suggestion
    ${chart.labelRange ? escapeXml/*xml*/ `<c:cat>${stringRef(chart.labelRange)}</c:cat>` : ""}
```
this will break excel export in Odoo due to a bug in the minifier that we use https://github.com/odoo/o-spreadsheet/pull/5863
We have to avoid nesting calls to escapeXml